app/poros/movie.rb

```diff
@@ -1,3 +1,9 @@
 class Movie
+  attr_reader :title, :rating, :id
 
-end
\ No newline at end of file
+  def initialize(attributes)
+    @title = attributes['title']
+    @rating = attributes['vote_average']
+    @id = attributes['id']
+  end
+end
```

Comment:
Recommend replacing with the poro I made, and adding checks to make sure atleast id, title, and rating work even when everything else is nil